js/source/tile.js

```diff
@@ -138,7 +138,12 @@ Tile.prototype = {
     },
 
     getElementGroups: function(layer, shaderName) {
-        return this.elementGroups && this.elementGroups[layer.ref || layer.id] && this.elementGroups[layer.ref || layer.id][shaderName];
+        if (!this.buffers || !Object.keys(this.buffers)) return null;
```

Comment:
Thank you. Pushing a fix shortly.

Side note: the `!Object.keys(this.buffers)` branch is total [cargo cult programming](https://en.wikipedia.org/wiki/Cargo_cult_programming). It is never triggered AFIK but I'm preserving it as a refactor of `hasLayerData`. Maybe that's a mistake and it should be removed altogether.